samples/app-complete-auth/nodejs/index.js

```diff
@@ -63,7 +63,8 @@ adapter.onTurnError = async (context, error) => {
     'https://www.botframework.com/schemas/error',
     'TurnError'
   );
-
+   // Uncomment this for local debugging.
+  //await context.sendActivity(`Sorry, it looks like something went wrong. Exception Caught: ${error}`);
```

Comment:
// await context.sendActivity(`Sorry, it looks like something went wrong. Exception Caught: ${error}`);